Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM registry.suse.com/bci/golang:1.21-openssl AS builder
+
+RUN set -euo pipefail; zypper -n  in --no-recommends git make ; zypper -n clean;
+
+# Create and change to the /var/lib/ directory.
+WORKDIR /var/lib
+
+# For now, we need this since we use replace directive to point to the local telemetry module in go.mod
+RUN git clone https://github.com/SUSE/telemetry
+RUN cd telemetry; make build
+
+# Copy local code to the container image.
+RUN mkdir ./telemetry-server
+COPY . ./telemetry-server
+
+RUN cd telemetry-server; make build
+
+FROM registry.suse.com/bci/bci-base:15.6
```

Comment:
I would put a comment here to help visually separate the builder phase from the final image.

Reply:
Updated